### ISSUE TITLE
feat: allow embedding of lform javascript/style questionnaire files using the setupHeader syntax

### ISSUE THREAD

**Issue by adunsulag** [openemr/openemr#7699]
It would be nice to be able to pull in the lform questionnaire files using the setupHeader() syntax in twig.  Currently we can include Questionnaire's using the lform_webcomponents.php file but we can't include that inside of our twig template files.  It would be ideal if we could pull in lform files using the twig function setupHeader.

This issue depends on the issue mentioned in #7698.  If we can configure a script file to be included as a module we can setup configurations for the two different style formats in the lform_webcomponents.php file like so:
```
    questionnaires:
        basePath: '%webroot%/interface/forms/questionnaire_assessments/'
        script:
            - src: lforms/webcomponent/assets/lib/zone.min.js
              type: module
            - src: lforms/webcomponent/runtime.js
              type: module
            - src: lforms/webcomponent/polyfills.js
              type: module
            - src: lforms/webcomponent/main.js
              type: module
            - src: lforms/fhir/R4/lformsFHIR.min.js
              type: module
        link: /assets/styles/styles_openemr_lforms.css
    questionnaires-lform:
        basePath: '%webroot%/interface/forms/questionnaire_assessments/'
        script:
            -   src: lforms/webcomponent/assets/lib/zone.min.js
                type: module
            -   src: lforms/webcomponent/runtime.js
                type: module
            -   src: lforms/webcomponent/polyfills.js
                type: module
            -   src: lforms/webcomponent/main.js
                type: module
            -   src: lforms/fhir/R4/lformsFHIR.min.js
                type: module
        link: /assets/styles/styles_openemr_lforms.css
```

@sjpadgett I don't think this will impact you at all, but you probably want to be aware of it.

**Comment by sjpadgett** [comment]
@adunsulag 
A good idea but when I first integrated lform web components I ran into issues with the includes and their scope on the form.
May or may not still be an issue or I simply implement wrongly in the first place.

We could also clone lform project to our openemr repo until that projects admin finish their npm integration.
`
A complete npm package that you can use with import/require statements in your code is not ready yet. The current lforms npm package only contains built files in /dist/lforms directory.
`

I'd be happy to do yhis.

**Comment by adunsulag** [comment]
@sjpadgett the big problem I ran into was I had to include it as a type="module".  Ran into the same scoping issue when I just did a regular type="text/javascript".  Due to the way they packaged their stuff, had to make this change.

I think what you've done has been working pretty well.  Not sure its needed to clone the project into OpenEMR but you'd probably the advantages of that better than I would.
